src/js/ripple/amount.js

```diff
@@ -975,29 +937,25 @@ Amount.prototype.not_equals_why = function(d, ignore_issuer) {
     return 'Not an Amount';
   }
 
-  var result = false;
-
   if (!this.is_valid() || !d.is_valid()) {
-    result = 'Invalid amount.';
+    return 'Invalid amount.';
```

Comment:
No need for `else` if you're using returns. Personally I like the old approach of using a `result` var that gets returned in the end, but early return is fine with me as well. If you're using returns though, don't use `else`'s. Makes it confusing